checkout_session.go

```diff
@@ -187,6 +240,21 @@ type CheckoutSessionListParams struct {
 	Subscription  *string `form:"subscription"`
 }
 
+// CheckoutSessionCustomerDetailsTaxIds represent customer's tax IDs at the
+// time of checkout.
+type CheckoutSessionCustomerDetailsTaxIds struct {
+	Type  CheckoutSessionCustomerDetailsTaxIdsType `json:"type"`
+	Value string                                   `json:"value"`
+}
+
+// CheckoutSessionCustomerDetails represent the customer details including
+// the tax exempt status and the customer's tax IDs.
+type CheckoutSessionCustomerDetails struct {
+	Email     string                                  `json:"email"`
+	TaxExempt CheckoutSessionCustomerDetailsTaxExempt `json:"tax_exempt"`
+	TaxIds    *[]CheckoutSessionCustomerDetailsTaxIds `json:"tax_ids"`
```

Comment:
Not sure about this. Need to double check that this should be a pointer

Reply:
Nah, I'd make it a non-pointer. All slices are pointers already in the way that they can hold a `nil` value (which behaves like an empty list).